### ISSUE TITLE
sst2	human AMSH/STAMBP protein  ubiquitin specific-protease

### ISSUE THREAD

**Issue by IanDMedeiros** [nextgenusfs/gene2product#17]
`sst2	human AMSH/STAMBP protein  ubiquitin specific-protease` is in the ncbi_cleaned_gene_products list, but it throws a fatal error in the discrepancy report:

```
FATAL: DiscRep_SUB:SUSPECT_PRODUCT_NAMES::Remove organism from product name
DiscRep_SUB:DISC_PRODUCT_NAME_QUICKFIX::1 features contains 'human'
```
Should this be removed from the list and get flagged for manual curation instead? (Also, seems like it may be a bad annotation to begin with?) I imagine the same might be true for five other products containing "human" in ncbi_cleaned_gene_products, but this is the only one I have encountered in my own data.

**Comment by hyphaltip** [comment]
Yeah a lot of these in there could be cleaned up. There is also a set of truncations happening in funannotate on the eggnog products. I think a better strategy could be adopted for product names pulling from the NCBI approved products file too.
